app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -123,7 +123,8 @@ class TopicLessonsFragmentTest {
   }
 
   private fun setUpTestApplicationComponent() {
-    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
```

Comment:
Let's revert the change in this line, as it is not required to explicitly mention class name for "this"

Reply:
@Sci3fic please reply to all comments.

Reply:
Done.